justfile

```diff
@@ -7,8 +7,27 @@ _default:
 build-backend:
   cd backend && cargo build --release
 
+build-frontend:
+  cd frontend && npm run build
+
+build: build-backend build-frontend
+
+build-and-run: build run-app
+
+build-flake-backend:
+  nix build .#game-night-backend
+
+build-flake-frontend:
+  nix build .#game-night-frontend
+
+build-flake-docker:
+  nix build .#game-night-docker
+  docker load < result
+
+build-flake: build-flake-backend build-flake-frontend build-flake-docker
```

Comment:
This is probably not needed, because flakes manage dependencies (so `nix build .#game-night-docker` will build `game-night-frontend` and `game-night-backend` if their inputs have changed)

I do like the addition of the other flake commands though

Reply:
Ah didn't know that. Yeah we can toss this one.